tapir/shifts/management/commands/apply_shift_cycle_start.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.db.models import Max
+
+from tapir.shifts.models import ShiftCycleEntry, ShiftTemplateGroup, Shift
+from tapir.utils.shortcuts import get_monday
+
+
+class Command(BaseCommand):
+    help = "If a new cycle has started, remove one shift point from all active members."
+
+    def handle(self, *args, **options):
+        if ShiftCycleEntry.objects.exists():
+            last_cycle_date: datetime.date = ShiftCycleEntry.objects.aggregate(
+                Max("cycle_start_date")
+            )["cycle_start_date__max"]
+            new_cycle_start_date = last_cycle_date + datetime.timedelta(
+                days=ShiftCycleEntry.SHIFT_CYCLE_DURATION
+            )
+        else:
+            new_cycle_start_date = self.get_first_cycle_start_date()
+
+        if new_cycle_start_date is None:
+            return
+
+        while datetime.date.today() >= new_cycle_start_date:
+            ShiftCycleEntry.apply_cycle_start(new_cycle_start_date)
+            new_cycle_start_date += datetime.timedelta(
+                days=ShiftCycleEntry.SHIFT_CYCLE_DURATION
+            )
+
+    @staticmethod
+    def get_first_cycle_start_date():
```

Comment:
I feel like this would be more elegantly solved by adding a `--date` argument to the command. This would make the first one a manual process. 

And I'm surprised, did we really draw shifts from people since the beginning? Color me surprised.

Reply:
Until now we never removed shift credits from anyone. You mean that we maybe didn't want to remove shift credits from the very first cycles but only from a later date?

I actually wanted to avoid manual first cycle on purpose. I thought it would be easy to forget when setting up a new Tapir instance. Do we have other similar things that must be manually done whenever a new Tapir comes?